triagebot.toml

```
@@ -1338,9 +1338,11 @@ compiletest = [
 "/src/doc/nomicon" =                                     ["@ehuss"]
 "/src/doc/reference" =                                   ["@ehuss"]
 "/src/doc/rust-by-example" =                             ["@ehuss"]
+"/src/doc/rustc" =                                       ["compiler", "@ehuss"]
 "/src/doc/rustc-dev-guide" =                             ["compiler"]
 "/src/doc/rustdoc" =                                     ["rustdoc"]
 "/src/doc/style-guide" =                                 ["style-team"]
+"/src/doc/unstable-book" =                               ["compiler", "@ehuss"]
```

I actually don't want this if the compiler team is willing to do it. I rarely have enough context on these features, and my approvals are often just rubber stamps.

```suggestion
"/src/doc/unstable-book" =                               ["compiler"]
```

---

Changed.